lib/pdk/cli/validate.rb

```diff
@@ -70,15 +71,47 @@ module PDK::CLI
 
       options = targets.empty? ? {} : { targets: targets }
       options[:auto_correct] = true if opts.key?(:'auto-correct')
-
-      exit_code = 0
+      # options[:parallel] = true if opts.key?(:parallel)
 
       # Ensure that the bundle is installed and tools are available before running any validations.
       PDK::Util::Bundler.ensure_bundle!
 
-      validators.each do |validator|
-        exit_code = validator.invoke(report, options)
-        break if exit_code != 0
+      mod_root = PDK::Util.module_root
+      unless mod_root
+        raise PDK::CLI::FatalError, _('Current working directory is not part of a module. (No metadata.json was found.)')
+      end
```

Comment:
I did not view this as a behavior change. previously we had this code around the console command call in cli/exec.rb. This had to be brought out to this level because Dir.chdir changes the working dir of the process, not the thread.